examples/api-routes-apollo-server-and-client/lib/init-apollo.js

```diff
@@ -0,0 +1,31 @@
+let _apolloClient
+
+const ssrMode = typeof window === 'undefined'
+
+export default function initApollo (clientFn, options) {
+  if (!clientFn) {
+    throw new Error(
+      '[withApollo] the first param is missing and is required to get the ApolloClient'
+    )
+  }
+
+  if (ssrMode) {
```

Comment:
This should be inlined, otherwise tree-shaking is not applied.

Reply:
@timneutkens sorry, I don't know what you're asking me to do. Can you please give an example?

Reply:
The typeof window part should be inside the if statement instead of assigning to a variable.